app/include/user_modules.h

```diff
@@ -59,6 +60,7 @@
 #define LUA_USE_MODULES_SPI
 //#define LUA_USE_MODULES_STRUCT
 //#define LUA_USE_MODULES_SWITEC
+//#define LUA_USE_MODULES_TCA9535
```

Comment:
First glimpse of the next module? 😄 
Please remove this line even though it's commented.